R/create_widget.R

```diff
@@ -0,0 +1,32 @@
+#' Converts an interactive ggplot2 object into an HTML widget
+#'
+#' @param   ggobj   An interactive \code{ggplot2} object.
+#' @param   width_svg,height_svg    Width and height for the output SVG.
+#' @param   sizing_width    Sets the width option in \code{ggiraph::opts_sizing}.
+#'
+#' @return  A \code{girafe} / \code{htmlwidget} object for use in a web browser.
+
+create_widget <- function(ggobj,
+                          width_svg,
+                          height_svg,
+                          sizing_width = 0.8) {
+  tooltip_css <- paste0(
```

Comment:
I'm happy to add this into {tfpscanner} for now, but this code will eventually be moved into {tfpbrowser} instead.